portal-api/src/v1/users/sign-in-link.service.js

```diff
@@ -90,6 +113,45 @@ class SignInLinkService {
       throw error;
     }
   }
+
+  async #incrementSignInLinkSendCount({ userId, userStatus, userSignInLinkSendDate, userEmail }) {
+    const MAX_SIGN_IN_LINK_SEND_COUNT = 3;
```

Comment:
It's a bit surprising that we're using screaming snake case for variables within methods like this